filebeat/autodiscover/builder/hints/config.go

```diff
@@ -26,7 +26,7 @@ type config struct {
 
 func defaultConfig() config {
 	rawCfg := map[string]interface{}{
-		"type": "docker",
+		"type": "container",
 		"containers": map[string]interface{}{
 			"paths": []string{
```

Comment:
Can we use a less docker-specific path here? Or add a `runtime` option that sets up the proper defaults depending on the runtime?

Reply:
I'm currently working on a different PR to make autodiscover more CRI friendly